windows/rngallery/rngallery.vcxproj

```diff
@@ -15,6 +16,7 @@
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
     <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18362.0</WindowsTargetPlatformVersion>
     <WindowsTargetPlatformMinVersion>10.0.16299.0</WindowsTargetPlatformMinVersion>
+    <PackageCertificatePassword>password</PackageCertificatePassword>
```

Comment:
What's this? I assume it has to do with the new pfx process, but maybe there needs to be a wiki/md update to explain this? Or a comment inline here?

Reply:
That line was also generated by `npx react-native-windows-init --version 0.xx-preview.x --overwrite` I believe. Let me test if I can remove it and see if the build still succeeds.

Reply:
Ok just tested. Looks like it isn't needed.